ecs_fargate/README.md

```diff
@@ -11,7 +11,8 @@ Get metrics from all your containers running in ECS Fargate:
 
 ### Installation
 
-Install the `dd-check-ecs_fargate` package manually or with your favorite configuration manager
+Install the `dd-check-ecs_fargate` package manually or with your favorite configuration manager.
+Additional details are available in our [AWS Fargate guide](https://docs.datadoghq.com/guides/awsfargate/).
```

Comment:
The link was updated. See https://github.com/DataDog/integrations-core/pull/1494